lib/MusicBrainz/Server/Form/Role/UserProfile.pm

```diff
@@ -53,11 +53,13 @@ has_field 'languages' => (
 
 has_field 'languages.language_id' => (
     type => 'Select',
+    messages => { required => N_l('A language is required') },
     required => 1
 );
 
 has_field 'languages.fluency' => (
     type => 'Select',
+    messages => { required => N_l('A fluency level is required') },
```

Comment:
Nitpick, but I'd prefer all error messages ended with periods:
```suggestion
    messages => { required => N_l('A language is required.') },
    required => 1
);

has_field 'languages.fluency' => (
    type => 'Select',
    messages => { required => N_l('A fluency level is required.') },
```

Reply:
I was copying User::Login, but I actually prefer this too, so changed it for these.